content/articles/changing-email.md

```diff
@@ -15,16 +15,6 @@ categories:
 
 ---
 
-## A note about accounts
```

Comment:
I think we can remove all of this if we agree to this: https://github.com/dnsimple/dnsimple-app/pull/27073/files#r1629893283

Reply:
👍 Agreed.